templates/sms.1.txt

```diff
@@ -0,0 +1 @@
+{{t "As requested, here is a link to install Firefox on your mobile device: %(link)s"}}
```

Comment:
I'm of two minds, though leaning on the side of "let's ship it to the l10n team." Even US-only, it'd be nice to have EN and ES, though, if we use short-codes, we might have to store [already translated strings with the SMS provider](https://dashboard.nexmo.com/buy-numbers/shortcodes/alerts#).